client/scripts/views/pages/members.ts

```diff
@@ -82,28 +163,46 @@ const MembersPage : m.Component<{}, { membersRequested: boolean, membersLoaded:
       m(Table, [
         m('tr', [
           m('th', 'Member'),
-          m('th.align-right', 'Posts/ Month'),
+          m('th.align-right', 'Posts / Month'),
         ]),
-        vnode.state.membersLoaded.map((info) => {
-          const profile = app.profiles.getProfile(info.chain, info.address);
-          return m('tr', [
-            m('td.members-item-info', [
-              m('a', {
-                href: `/${app.activeId()}/account/${info.address}?base=${info.chain}`,
-                onclick: (e) => {
-                  e.preventDefault();
-                  localStorage[`${app.activeId()}-members-scrollY`] = window.scrollY;
-                  navigateToSubpage(`/account/${info.address}?base=${info.chain}`);
-                }
-              }, [
-                m(User, { user: profile, showRole: true }),
+        vnode.state.profilesLoaded.map((profileInfo) => {
+            return m('tr', [
+              m('td.members-item-info', [
+                m('a', {
+                  href: `/${app.activeId()}/account/${profileInfo.profile.address}?base=${profileInfo.profile.chain}`,
+                  onclick: (e) => {
+                    e.preventDefault();
+                    localStorage[`${app.activeId()}-members-scrollY`] = window.scrollY;
+                    localStorage[`${app.activeId()}-members-numProfilesAlreadyLoaded`] = vnode.state.numProfilesLoaded;
+                    navigateToSubpage(`/account/${profileInfo.profile.address}?base=${profileInfo.profile.chain}`);
+                  }
+                }, [
+                  m(User, { user: profileInfo.profile, showRole: true }),
+                ]),
               ]),
-            ]),
-            m('td.align-right', info.count),
-          ]);
+              m('td.align-right', profileInfo.postCount),
+            ]);
         })]),
-    ]);
+        m('div', [
+          m('#infinite-scroll-wrapper', [
+            vnode.state.profilesFinishedLoading ? 
+            m('.infinite-scroll-reached-end', [
```

Comment:
@adamlawrencium Is this the best convention for UI hierarchies? Regarding class names and the use of different wrappers? Seems alright by me, but we should decide if there's a better pattern to use going forward if not!

Reply:
The use of an id here was actually part of a solution I since abandoned. Changing it back to the usual class name structure, not sure if you are referring to something else.